sourcegraph/about-replatform
# Add use case pages

Closes #41 by adding the new use cases pages and porting over relevant components.

NB: `CustomerLogosSectionAnimated` got some refactoring to play nicely with our linting, but this isn't the major overhaul planned for #70. Same thing with the carousels.

## To Test
- Check out the branch and look at the pages under `use-cases` (index, `onboarding`, & `vulnerabilities`).
- Ensure everything works as expected on desktop & mobile
